sdk/cmake/baseaddress_msvc.cmake

```diff
@@ -14,7 +14,7 @@ set(baseaddress_shlwapi                0x7c350000)
 set(baseaddress_rpcrt4                 0x7c2c0000)
 set(baseaddress_comctl32               0x7c1a0000)
 set(baseaddress_ole32                  0x7c060000)
-set(baseaddress_winspool.drv           0x7c030000)
+set(baseaddress_winspool               0x7c030000)
```

Comment:
Yes, it's the target name. winspool's target is just "winspool", other files (e.g. winmp3.acm) have the extension in the target name. So whatever the target does counts.

Reply:
Oh yes true @ThFabba . Then what about these lines:
```
dll/win32/setupapi/CMakeLists.txt	
  44 add_delay_importlibs(setupapi comdlg32 shell32 winspool.drv wintrust)
dll/win32/shell32/CMakeLists.txt	
 111 add_delay_importlibs(shell32 powrprof shdocvw devmgr winspool.drv winmm mpr uxtheme ole32 oleaut32 userenv browseui version fmifs)
```
They mention "winspool.drv" instead?

Reply:
The winspool cmakelists is a bit conflicting... it has `spec2def(winspool.drv winspool.spec ADD_IMPORTLIB)` but also `set_module_type(winspool win32dll UNICODE)`, was it renamed at some point?

Reply:
First parameter of spec2def is the dllname. This needs to be "winspool.drv", because this is the final filename. The rest usually refers to module names, where "winspool" is correct.
You are free to change everything to "winspool.drv" though in order to make it less confusing.